packages/pyodide-kernel-extension/src/index.ts

```diff
@@ -62,6 +68,28 @@ const kernel: JupyterFrontEndPlugin<void> = {
       }
     }
 
+    const logger = (options: { payload: ILogPayload; kernelId: string }) => {
+      if (!notebookTracker || !loggerRegistry) {
+        // nothing to do in this case
+        // TODO: simply log to the console?
+        return;
+      }
+
+      const { payload, kernelId } = options;
+
+      const notebook = notebookTracker.find(
+        (nb) => nb.sessionContext.session?.kernel?.id === kernelId,
+      );
+
+      if (!notebook) {
+        // no notebook found
+        return;
+      }
+
+      const logger = loggerRegistry.getLogger(notebook.sessionContext.path);
+      logger.log(payload);
```

Comment:
These logs are reported as `info`, which will not make the log console status bar item flash by default since it defaults to `warning`.

Some users have reported the flash to be irritating: https://github.com/jupyterlab/jupyterlab/issues/11225. So we can let users choose their log level and not default to `info`.